src/components/message/message-card-list/MessagesVirtualizedList.tsx

```diff
@@ -91,6 +101,29 @@ const MessagesVirtualizedList = (props: Props) => {
 		firstNextChunkIsLoaded,
 	]);
 
+	React.useEffect(() => {
+		toJS(messages).forEach(message =>
```

Comment:
why do you need do convert it toJS ? we have shallow observable, so messages inside should be plain objects

Reply:
It's strange, but when I tested before it just didn't work right, so I used toJS, ok I removed it